config/buildAssets.js

```diff
@@ -1,5 +1,9 @@
 /* eslint no-restricted-syntax: 0 no-shadow: 0 */
```

Comment:
Sorry, I forgot there are already eslint directives at the top of this file. We can add to that and use alternative syntax too:
```suggestion
/* eslint no-restricted-syntax: 0 no-shadow: 0 no-console: 0 */
```